Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallDiffCommand.cs

```diff
@@ -537,7 +531,12 @@ private void AddDirectory(string dirName, CancellationToken cancellationToken)
             _logger.LogDebug("Diff modified files processed.");
         }
 
-
+        private long GetTotalSizeModifiedFiles()
+        {
+            var modifiedFiles = _diffSummary.ModifiedFiles.Where(f => !_diffSummary.UnchangedFiles.Contains(f));
```

Comment:
do not use `var` is resulting type is unclear

Reply:
Done